tests/test_http_notebook.py

```diff
@@ -58,6 +58,7 @@ def test_http_notebook_01(self):
             results,
             {
                 'kind': 'results',
+                'protocol_version': [0, 11],
```

Comment:
Can you try this?

```suggestion
                'protocol_version': [0, ...],
```

would hate to update this test every time we bump the proto version

Reply:
What should the ellipsis do here? It just shows an error that ellipsis isn't equal to 11.

Reply:
Could we use HTTP headers instead of embedding in response body?

Reply:
Yeah, using a header makes sense. So `EdgeDB-Protocol-Version: 0.11`? (Apparently `X-` prefix shouldn't be used anymore https://stackoverflow.com/questions/3561381/custom-http-headers-naming-conventions)

Reply:
Also, @jaclarke, any issues with using a header?

@elprans can a custom HTTP header be stripped by some proxy? We need a protocol version to decode data, so this isn't some auxiliary stuff.

Reply:
Stripping random response headers is super bad mojo, so I wouldn't worry about that.

Reply:
> Also, @jaclarke, any issues with using a header?

Not that I can see, other than we need to send an extra CORS header to be able to access the custom header in the browser, but that's not really a problem. (https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_response_header)